### ISSUE TITLE
Apple Music is not available in your country or region (fresh 2.5.0 install) [BUG] 

### ISSUE THREAD

**Issue by ryanfantus** [Apple-Music-Electron/Apple-Music-Electron#187]
**Describe the bug**
After authenticating with my Apple account, I receive the message that Apple Music is not available in my country or region.

**To Reproduce**
1. Launch application, version 2.5.0
2. Attempt to login and authenticate
3. See error message and be forcibly logged out of application

**Expected behavior**
Continue any login flow and utilize application

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Linux - up-to-date 64 bit Arch
 - Application Version - 2.5.0 (apple-music-electron 2.5.0-1)

**Log File**
Log file mentions issues with launching additional Discord instance, which is curious since I do have Discord open but this application (while still being electron) is a different application.

**Additional context**
None

**Have you followed the troubleshooting guide found [here](https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Troubleshooting) and checked for any outstanding/pinned issues that can help?**
Yes

**Comment by coredev-uk** [comment]
How are you launching the application? Are you using a compiled state and what format did you use to install? Also, try following the [clean installation guide](https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Troubleshooting#clean-installation), this might fix your issue.

**Comment by ryanfantus** [comment]
Yep, tried the clean install guide. I can turn on authMode but then the GUI doesn't appear when I try to re-launch the application. I've launched from the .desktop file as well as directly from a CLI (using `apple-music-electron`).

**Comment by coredev-uk** [comment]
If you can turn on authMode, you are not on 2.5.0

**Comment by ryanfantus** [comment]
I used the following as a replacement PKGBUILD on arch to build it: https://pastebin.com/M7yXkPyA - however I just followed the install guide again and am doing another clean build.

The resultant `apple-music-electron` binary in the `dist/linux-unpacked` won't display any sort of gui. I'm now trying to redo it all using npm. It's throwing an error about missing an app-update.yml, but I'll let you know when it gets rebuilt via npm and if I have the same issues.

**Comment by ryanfantus** [comment]
Same problem. Ugh. Seems to be some weird electron stuff happening.

If I build with the pastebin pkgbuild, it "works" but I can't login, and I can still use authMode.

If I build it manually, I get no electron window and the following error messages:

1. Error: Error: ENOENT: no such file or directory, open '/home/ryan/Development/apple-music-electron/Apple-Music-Electron/dist/linux-unpacked/resources/app-update.yml'
2. [61960:0822/150116.801588:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.

Interestingly hitting https://music.apple.com/beta in Chrome directly seems to work without issues.

**Comment by coredev-uk** [comment]
The `app-update.yml` file error is common and should not affect application performance. What I don't understand is how you can still use authMode. What I suggest you do is create a fresh clone of AME and compile it into a pacman file and upgrade the existing pacman package.

**Comment by FordMustang0288** [comment]
I downloaded for windows, launched the EXE and was presented with this same error upon trying to login.

**Comment by coredev-uk** [comment]
> 
> 
> I downloaded for windows, launched the EXE and was presented with this same error upon trying to login.
> 
> EDIT:
> https://github.com/Apple-Music-Electron/Apple-Music-Electron/releases/tag/v2.5.0
> ...fixes the issue.
> 
> **However**, the UI is missing a lot of graphics.

Delete the cache folder for the application and that should fix it. Find more at https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Troubleshooting#clean-installation

**Comment by FordMustang0288** [comment]
> > I downloaded for windows, launched the EXE and was presented with this same error upon trying to login.
> > EDIT:
> > https://github.com/Apple-Music-Electron/Apple-Music-Electron/releases/tag/v2.5.0
> > ...fixes the issue.
> > **However**, the UI is missing a lot of graphics.
> 
> Delete the cache folder for the application and that should fix it. Find more at https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Troubleshooting#clean-installation

Fantastic! Sorry, I did see that but I figured it was just the fact it was beta. Super fast and helpful reply. Thanks!


Is it recommended that I follow this with a local copy of the repository or reinstall EXEs from time to time?

**Comment by coredev-uk** [comment]
> > > I downloaded for windows, launched the EXE and was presented with this same error upon trying to login.
> > > EDIT:
> > > https://github.com/Apple-Music-Electron/Apple-Music-Electron/releases/tag/v2.5.0
> > > ...fixes the issue.
> > > **However**, the UI is missing a lot of graphics.
> > 
> > 
> > Delete the cache folder for the application and that should fix it. Find more at https://github.com/Apple-Music-Electron/Apple-Music-Electron/wiki/Troubleshooting#clean-installation
> 
> Fantastic! Sorry, I did see that but I figured it was just the fact it was beta. Super fast and helpful reply. Thanks!
> 
> Is it recommended that I follow this with a local copy of the repository or reinstall EXEs from time to time?

Just do a fresh install from the repo and then all updates should be handled by the application's auto updater.

**Comment by ryanfantus** [comment]
Hey, I was able to get things to build in linux. I installed 2.5.0 from the releases beta, and then re-built everything off the latest cut of master and overwrote what's in `/opt/apple-music-electron/` with the contents of my `dist/linux-unpacked` directory. Everything seems to work now. Perhaps the arch aur would just benefit from an `apple-music-electron-git` package :) I'll see what it takes to do that, why not.
